src/sections/MainSection.js

```diff
@@ -0,0 +1,78 @@
+import styled from "styled-components";
+import homeVideo from "../assets/home.mp4";
+import Button from "../components/Button";
+
+const MainSection = () => (
+  <SectionContainer>
+    <video
+      style={styles.video}
+      src={homeVideo}
+      autoPlay={true}
+      loop={true}
+      muted={true}
+    />
+    <OverlayContent>
+      <Title>KANEKI RAMOS</Title>
+      <Details>
+        Full Stack Developer / Game Designer / Software Engineer
+      </Details>
+      <ButtonWrapper>
+        <Button style={styles.Button}>Check me out</Button>
+        <Button style={styles.Button}>Get in contact</Button>
+      </ButtonWrapper>
+    </OverlayContent>
+  </SectionContainer>
+);
+
+const styles = {
+  video: {
+    objectFit: "cover",
+    height: "100vh",
+    width: "100vw",
+  },
+
+  Button: {
+    border: "2px solid",
+    padding: "5px 24px 16px 24px",
+    backgroundColor: "transparent",
+    borderRadius: "5px",
+    color: "white",
+    cursor: "pointer",
+    transition: "all 0.3s ease-out",
+  },
+};
+
+const Title = styled.h1`
+  font-size: 105px;
+  margin-top: -250px;
```

Comment:
negative margins are a hack and a code smell, this is no bueno, and will give you a lot of problems specially in different browsers and mobile when you do react native for example.  **Avoid negative positioning and spacing at all cost**